database/migrations/2023_11_14_122252_add_type_column_to_collections_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Enums\TokenType;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('collections', function (Blueprint $table) {
+            $table->string('type')->default(TokenType::Erc721->value);
```

Comment:
Should you add index for this column and it is being used in `where` conditions?

Reply:
Can't tell for sure, I will conduct experiments on the larger `nfts` table to see where we execute queries to get NFTs, to see if we gain some performance.

Reply:
any update on this?